language-support/hs/bindings/BUILD.bazel

```diff
@@ -80,6 +86,9 @@ da_haskell_test(
     ],
     main_function = "DA.Ledger.Tests.main",
     src_strip_prefix = "test",
+    # We spin up Sandbox as a separate process, so
+    # try not to overload the machine.
+    tags = ["cpu:2"],
```

Comment:
Should these tests run with `"exclusive"`?

Reply:
Good question, I don’t know tbh. Exclusive makes everything slower since nothing else can run in parallel. Of course if things run in parallel but they fail then things might get slower. Locally, this reduces most flakes for me so I think this is probably faster than `exclusive` but we can revisit if it still is super flaky.